partials/kogito-extensions.adoc

```diff
@@ -0,0 +1,39 @@
+.Kogito Extensions
+[cols="1,2,3a"]
+|===
+| Extension Name | Description | Maven Snippet
+ 
+| kogito-quarkus | Adds support for Quarkus | 
+[source,xml]
+----
+<dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-quarkus</artifactId>
+</dependency>
+----
+
+| kogito-quarkus-rules | Adds Quarkus support for Drools Rule Language (DRL) | 
+[source,xml]
+----
+<dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-quarkus</artifactId>
```

Comment:
```suggestion
    <artifactId>kogito-quarkus-rules</artifactId>
```

Reply:
Is this a change to the artifact name, or an additional one we need to add?

Reply:
well, not expert in our kogito extensions, I think this is just a copy paste issue.

if I have a look on table:
```
.Kogito Extensions
[cols="1,2,3a"]
```

its first columns says extension name, that is different for the three rows, so I think  each row should point to different  maven artifact, I think three maven artifacts that the table should mention are:
- `kogito-quarkus`
- `kogito-quarkus-rules`
- `kogito-quarkus-decisions`